modules/fluid_properties/include/userobjects/SinglePhaseFluidProperties.h

```diff
@@ -56,7 +56,7 @@ class SinglePhaseFluidProperties : public FluidProperties
    *
    * @param[in] v   specific volume
    * @param[in] e   specific internal energy
-   * @returns sound speed
+   * @return sound speed
```

Comment:
If you just want consistent style with the rest of the file, that's fine, but I think `@return` and `@returns` are both valid Doxygen commands.

Reply:
You are right. I thought they only had `@return`